packages/mm-backend/src/common/enums/memo.enum.ts

```diff
@@ -0,0 +1,36 @@
+export enum TradingType {
```

Comment:
What's that?

Reply:
It's excluded to another branch: Fork-issue_17-snapshot-processing